examples/multi-vpp/multi_vpp.go

```diff
@@ -58,10 +58,10 @@ func main() {
 
 	// since sockets default to the same value
 	if *binapiSockAddrVpp1 == *binapiSockAddrVpp2 {
-		log.Fatalln("ERROR: identical VPP binapi sockets defined, set at least one of them to a non-default path")
+		log.Println("ERROR: identical VPP binapi sockets defined, set at least one of them to a non-default path")
```

Comment:
Is this intent? Shouldn't the test end here?

Reply:
It will fail the example then when running `TestExamples`. It still connects two times just to same VPP. In the future we might improve the test case and run two VPP instances.